docs/serving/samples/cloudevents/cloudevents-nodejs/index.js

```diff
@@ -0,0 +1,87 @@
+const express = require('express')
+const { HTTPReceiver } = require('cloudevents-sdk')
+const { CloudEvent, StructuredHTTPEmitter } = require('cloudevents-sdk/v1')
+const Constants = require('cloudevents-sdk/lib/bindings/http/constants.js')
+
+const target = process.env.K_SINK
+const app = express()
+const receiver = new HTTPReceiver()
+
+const main = () => {
+  app.listen(8080, function () {
+    console.log('Cookie monster is hungry for some cloudevents on port 8080!')
+  })
+}
+
+// handle shared the logic for producing the Response event from the Request.
+const handle = (data) => {
+  return { message: `Hello, ${data.name}` }
+}
+
+// receiveAndSend is invoked whenever we receive an event.
+const receiveAndSend = (cloudEvent, res) => {
+  let myEvent = new CloudEvent()
+    .type('dev.knative.docs.sample')
+    .source('https://github.com/knative/docs/docs/serving/samples/cloudevents/cloudevents-nodejs')
+    .time(new Date())
+  const emitter = new StructuredHTTPEmitter({
+    method: "POST",
+    url: target
+  });
+  res.status(201).end()
+  emitter.emit(myEvent)
+}
```

Comment:
This should eventually be something like:

```js
const emitter = new HTTPEmitter();
emitter.emit({ url: target }, myEvent, "structured");
```

Reply:
Using HTTPBinary now to match Go sample
Hope HTTPBinary.emmiter also has a send()